### ISSUE TITLE
newpop

### ISSUE THREAD

**Issue by sena1989** [sena1989/angularjslogin1#6]
https://stackoverflow.com/questions/36542768/ng-repeat-in-nested-array-in-angularjs

https://stackoverflow.com/questions/23264789/angularjs-ng-repeat-to-assign-generate-a-new-unique-id

<!DOCTYPE html>
<html>

  <head>
    <script data-require="jquery@2.0.0" data-semver="2.0.0" src="//ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js"></script>
	<link rel="stylesheet" href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.min.css">
    <link data-require="bootstrap-css@3.0.3" data-semver="3.0.3" rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css" />
    
    <script data-require="ui-bootstrap@0.10.0" data-semver="0.10.0" src="http://angular-ui.github.io/bootstrap/ui-bootstrap-tpls-0.10.0.js"></script> 
	<link data-require="bootstrap@3.3.7" data-semver="3.3.7" rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
    <link data-require="bootstrap-css@3.*" data-semver="3.3.7" rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.css" />
    
	<script data-require="angular.js@1.5.6" data-semver="1.5.6" src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.6/angular.min.js"></script>
    <script data-require="angular-animate@1.6.*" data-semver="1.6.5" src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.6.5/angular-animate.min.js"></script>
    <script data-require="angular-touch@1.6.*" data-semver="1.6.2" src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.2/angular-touch.js"></script>
    <script data-require="ui-bootstrap@*" data-semver="2.5.0" src="https://cdn.rawgit.com/angular-ui/bootstrap/gh-pages/ui-bootstrap-tpls-2.5.0.js"></script> 
    
    <link rel="stylesheet" href="style.css" />
    <script src="script.js"></script>
  </head>

  <body ng-app="app" ng-controller="ModalCtrl">
    <div ng-form="Form">
      <ul class="radio-tab">
	  
	
	
		 <div class="col-md-12 col-sm-12 col-xs-12 form-group tab-title">
		          

				  
				  <div class="custom-radio">
  <label><input type="radio" class="radio-inline" ng-model="selection" ng-value="'first'" name="selection" ng-required="'true'"><span class="outside">
  <span class="inside"></span></span>Option One</label>
  <label><input type="radio"class="radio-inline" ng-model="selection" ng-value="'second'" name="selection" ng-required="'true'"><span class="outside">
  <span class="inside"></span></span>Option Two</label>
</div>
				  
					

				
		  </div>
	

				
	     <div class="">
		 
            
            
          
		 
            
            
          
		 </div>
        <li class="col-xs-6 tab-list-item">
          
            
            
          
		<div disable-children="selection != 'first'">  
		    <form class="form-horizontal selection-tab" role="form" ng-submit="submit()" name="myForm">
				<div class="form-group">
				  <label class="control-label col-sm-4" for="email">Name</label>
				  <div class="col-sm-8">
					
					<input type="text" class="form-control" ng-model="ekaText" name="ekaText" ng-pattern="/^[a-zA-Z]+$/" ng-required="'true'" ng-disabled="selection != 'first'" required/>
				  </div>	  
				</div>		  
				
				<div class="form-group">
				  <label class="control-label col-sm-4" for="email">Email</label>
				  <div class="col-sm-8">
					<input type="text" class="form-control" ng-model="ekaText" name="ekaText" ng-pattern="/^[a-zA-Z]+$/" ng-required="'true'" ng-disabled="selection != 'first'" required/>
				  </div>	  		 
				</div>
			 
				<div class="form-group">
				  <label class="control-label col-sm-4" for="email">Country</label>
				  <div class="col-sm-8">
					<input type="text" class="form-control" ng-model="ekaText" name="ekaText" ng-pattern="/^[a-zA-Z]+$/" ng-required="'true'" ng-disabled="selection != 'first'" required/>
				  </div>		  
				</div>
				
				<div class="form-group">
				  <label class="control-label col-sm-4" for="pwd">Password</label>
				  <div class="col-sm-8">
					<input type="text" class="form-control" ng-model="ekaText" name="ekaText" ng-pattern="/^[a-zA-Z]+$/" ng-required="'true'" ng-disabled="selection != 'first'" required/>
				  </div>	  
				</div>
			   
				<div class="form-group">
				  <div class="col-sm-offset-2 col-sm-10">
					<button type="submit" class="btn btn-success custom-width pull-right" ng-click="open()" ng-disabled="selection != 'first'">Submit</button>		
				  </div>
				</div>
			</form>
	    </div>
    </li>
		
		
		
        <li class="col-xs-6 tab-list-item">
          
            

            
          
          <div disable-children="selection != 'second'">
		  <form class="form-horizontal selection-tab" role="form" ng-submit="submit()" name="myForm">
            <div class="form-group">
      <label class="control-label col-sm-4" for="email">Country:</label>
      <div class="col-sm-8">
		<input type="file" class="form-control" ng-model="ekaText" name="ekaText" ng-pattern="/^[a-zA-Z]+$/" ng-required="'true'" ng-disabled="selection != 'second'" />
      
	  </div>		  
    </div>
	</div>
			
          </div>
        </li>
      </ul>
     
    </div>
  </body>

</html>
------
var app = angular.module('app', ['ui.bootstrap']);

app.controller('ModalCtrl', function($scope, $uibModal) {

  $scope.open = function() {
    var modalInstance =  $uibModal.open({
      templateUrl: "modal-form.html",
      controller: "ModalContentCtrl",
      size: '',
    });
    
    modalInstance.result.then(function(response){
        $scope.result = `${response} button hitted`;
    });
    
  };
  
})

app.controller('ModalContentCtrl', function($scope, $uibModalInstance) {

  $scope.ok = function(){
    $uibModalInstance.close("Ok");
  }
   
  $scope.cancel = function(){
    $uibModalInstance.dismiss();
  } 
  
});
